core/src/main/java/org/elasticsearch/repositories/fs/FsRepository.java

```diff
@@ -54,9 +54,9 @@
     public static final Setting<String> REPOSITORIES_LOCATION_SETTING =
         new Setting<>("repositories.fs.location", LOCATION_SETTING, Function.identity(), Property.NodeScope);
     public static final Setting<ByteSizeValue> CHUNK_SIZE_SETTING =
-        Setting.byteSizeSetting("chunk_size", "-1", Property.NodeScope);
+        Setting.memorySizeSetting("chunk_size", "-1", Property.NodeScope);
     public static final Setting<ByteSizeValue> REPOSITORIES_CHUNK_SIZE_SETTING =
-        Setting.byteSizeSetting("repositories.fs.chunk_size", "-1", Property.NodeScope);
+        Setting.memorySizeSetting("repositories.fs.chunk_size", "-1", Property.NodeScope);
```

Comment:
Should these two settings have the ability to specify as a percentage of heap as well as absolute bytes?